src/coreclr/inc/CrstTypeTool.cs

```diff
@@ -64,9 +64,9 @@ public static int Main()
         {
             // Calculate the filenames of the input and output files.
             string inputFile = "CrstTypes.def";
-            string outputFile = "crsttypes.h";
+            string outputFile = "crsttypes_generated.h";
```

Comment:
Nit: I would make the capitalization of `CrstTypes` consistent while you are touching this.

Reply:
So you would like to see `"CrstTypes_generated.h"`?

Reply:
Hmm, we are very inconsistent with casing. I guess it can stay as is.